plugins/grpc_streamer_plugin/ufm_sim_web_service/Subscriber.py

```diff
@@ -0,0 +1,218 @@
+#
+# Copyright © 2013-2022 NVIDIA CORPORATION & AFFILIATES. ALL RIGHTS RESERVED.
+#
+# This software product is a proprietary product of Nvidia Corporation and its affiliates
+# (the "Company") and all right, title, and interest in and to the software
+# product, including all associated intellectual property rights, are and
+# shall remain exclusively with the Company.
+#
+# This software product is governed by the End User License Agreement
+# provided with the software product.
+#
+from google.protobuf.any_pb2 import Any
+import google.protobuf.timestamp_pb2
+import requests
+import time
+from datetime import datetime
+from threading import Lock
+import grpc_plugin_streamer_pb2 as grpc_plugin_streamer_pb2
+from Config import RESTCall,Constants
+
+
+class Subscriber:
+    """
+    For manage the client job, request get calls and threads
+    """
+    job_id_messages = 1
+    lock = Lock()
+
+    def __init__(self, ip, rest_api_calls, session, host):
+        """
+        :param ip: name/ip of machine that asked this information
+        :param rest_api_calls: list of string that represent the wanted calls
```

Comment:
not clear what is the format of the rest_api_calls parameter. is it comman separated? please add info...